frontend/.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+  "extends": [
+    "react-app",
+    "plugin:jsx-a11y/recommended",
+    "prettier",
+    "prettier/react"
+  ],
```

Comment:
It doesn't seem to complain to me when I make bad formatting mistakes like indentation, multiple semicolons, missing semicolons... am I doing something wrong?

Both `npm run start` and `npm run build` doesn't complain for me.

Also, thoughts on using the [airbnb](https://github.com/airbnb/javascript) style guide for eslint? We were discussing this during the last meeting. It's quite opinionated, which I think is helpful to stay consistent in a large team without having to nitpick during pull requests :)

Reply:
The eslint when run from `npm run start` and `npm run build` does not seem to adhere to the `.eslintrc` file? I just ran `./node_modules/eslint/bin/eslint.js` directly on a file and it produced better results - results that seem to match the `.eslintrc`.